### ISSUE TITLE
Add option to configure registry mirrors

### ISSUE THREAD

**Issue by mayrstefan** [rancher-sandbox/rancher-desktop#719]
To avoid Docker Hub pull limits or work with other infrastructure restrictions it would be necessary to configure a registry mirror. Especially since containerd has way more [advanced options](https://github.com/containerd/containerd/blob/main/docs/cri/registry.md) to configure configure mirrors for multiple registries.

To make use of this feature we need a way to modify or extend the containerd configuration. Or at least a documentation how this can be done with Rancher Desktop.

**Comment by xiaods** [comment]
same here. the scenario is use nerdctl build image, I need setting local registry mirror config. i know docker registry have config for this requirements but rancher desktop not this.

**Comment by janeczku** [comment]
The registry mirror configuration should also be applied to both k3s/containerd as well as docker/nerdctl CLI operation.

**Comment by xiaods** [comment]
@janeczku  I need a UI to gudie the setup. don't need know the config file.

**Comment by debanjanbasu** [comment]
Indeed, not even sure where is the config file located, and how to even update it. Rancher-desktop, is all about the UI experience, I presumed.
